index.js

```diff
@@ -523,6 +546,11 @@
       this === other;
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this <= other;
+  }
+
```

Comment:
My comment was inaccurate. I'd forgotten the details of #47. What I suggest is that we update the `lte` function to check `typeof x === typeof y` in addition to `type(x) === type(y)`. We'll then be able to update `Boolean$prototype$lte`, `Number$prototype$lte`, and `String$prototype$lte`.